operators/src/plot/histogram.rs

```diff
@@ -264,10 +267,12 @@ impl HistogramRasterQueryProcessor {
             let mut computed_metadata = HistogramMetadataInProgress::default();
 
             while let Some(tile) = input.next().await {
-                let tile = tile?;
-
-                computed_metadata
-                    .add_raster_batch(&tile.grid_array.data, tile.grid_array.no_data_value);
+                match tile?.grid_array {
+                    geoengine_datatypes::raster::GridOrEmpty::Grid(g) => {
+                        computed_metadata.add_raster_batch(&g.data, g.no_data_value);
+                    }
+                    geoengine_datatypes::raster::GridOrEmpty::Empty(_) => {} // TODO: find out if we really do nothing for empty tiles?
```

Comment:
I guess we need to increase no data value counts and you already did implement a method for that.

Reply:
the add_raster_batch does this:
```
                if v == no_data {
                    continue;
                }
```